lang/en/tool_cloudmetrics.php

```diff
@@ -85,4 +85,8 @@
 
 // Error messages.
 $string['metric_not_found'] = 'Metric \'{$a}\' not found';
-
+$string['collector_failed'] = 'Metric \'{$a}\' failed %s';
+$string['collector_passed'] = 'Metric \'{$a}\' succeeded';
+$string['no_issues'] = 'No issues';
```

Comment:
I think for a check this simple you can probably just put the verbose details straight into the summary. So instead of 'No issues' it would say roughly 'db:ok, cloudwatch: ok' (with formatting etc)

![image](https://user-images.githubusercontent.com/187449/167739721-45e7a703-dab0-46a6-972b-c6de6e974bd5.png)